src/pybind/mgr/dashboard_v2/frontend/src/app/ceph/cluster/hosts/hosts.component.ts

```diff
@@ -41,13 +41,6 @@ export class HostsComponent implements OnInit, OnDestroy {
         pipe: this.cephShortVersionPipe
       }
     ];
-    this.interval = setInterval(() => {
```

Comment:
Some components are still refreshing data manually:

- pool-detail.component
- clients.component
- cephfs.component

Can you also adapt this components to use the datatable auto refresh feature?

Reply:
done